src/svg/Connecting.svg

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" style="display: block; shape-rendering: auto;" viewBox="0 0 100 100" preserveAspectRatio="xMidYMid">
```

Comment:
Please add a comment documenting the source of the spinner:

    <!--
    Spinner from: https://loading.io/spinner/spinner/-spinner-preloader-ajax-loading-icon
    License: https://loading.io/license/#free-license
    -->